stdlib/multiprocessing/heap.pyi

```diff
@@ -0,0 +1,32 @@
+import sys
+from _typeshed import Incomplete
+from collections.abc import Callable
+from mmap import mmap
+from typing_extensions import TypeAlias
+
+_Block: TypeAlias = tuple[Arena, int, int]
+
+__all__ = ["BufferWrapper"]
+
+class Arena:
+    size: int
+    buffer: mmap
+    if sys.platform == "win32":
+        name: str
+        def __init__(self, size: int) -> None: ...
+    else:
+        fd: int
+        def __init__(self, size: int, fd: int = ...) -> None: ...
+
+if not sys.platform == "win32":
```

Comment:
Interesting, why this choice? This seems to go against most recommendations for python comparisons I've seen

Reply:
It does? I haven't seen those recommendations, and I'd be curious to know why they recommend against using the `!=` operator :)

Anyway, this is the standard typeshed style for this kind of guard :)

Reply:
No worries, sounds good. I'll look if I can find the related documentation I'm thinking of about the application of de morgan's laws in relationship to the `not` and `!=`/`==` operators.

Reply:
De Morgan's laws are meaningful when you have `not` in front of `and` or `or`. For example,
```python3
if not (foo == 0 or bar == 0):
    print("nothing is zero")
```
is (by De Morgan's law) same as
```python3
if foo != 0 and bar != 0:
    print("nothing is zero")
```
The first way to write this can be cleaner with two or more things to check, here `foo` and `bar`. You only have one thing, `sys.platform`.